cosim-demo-app/user-guide.md

```diff
@@ -250,54 +250,23 @@ Note: The new OSP-IS connection types are not supported when using the SSP stand
 
 [back to top](./user-guide#user-guide)
 
-## Distributed co-simulation using fmu-proxy
+## Distributed co-simulation using proxyfmu
 
-Distributed simulation using fmu-proxy allows you to run multiple instances of a model, run models that are not compatible
-with your operational system or to parallelize the workload onto multiple computation nodes. See more details on [fmu-proxy.](../libcosim/fmuproxy)
+Distributed simulation using proxyfmu allows you to run multiple instances of a model, run models that are not compatible
+with your operational system or to parallelize the workload onto multiple computation nodes. See more details on [proxyfmu.](../libcosim/distributed)
 
-The demo case dp-ship has a specific configuration to be used with fmu-proxy (see the file “OspSystemStructure.xml” or “SystemStructure.ssd” 
-under the folder *\fmuproxy*). Note that the only difference when comparing to the regular configuration is related to the source of the FMU file. 
+The demo case dp-ship has a specific configuration to be used with proxyfmu (see the file “OspSystemStructure.xml” or “SystemStructure.ssd” 
+under the folder *\proxyfmu*). Note that the only difference when comparing to the regular configuration is related to the source of the FMU file. 
 
-There are three different ways to specify the FMU source to be loaded by the fmu-proxy. Below is an example for the "OspSystemStructure.xml":
+Below is an example of how to specify proxyfmu sources for the "OspSystemStructure.xml":
 ```xml
 <Simulators>
-    <Simulator name="FMU1" source="fmu-proxy://localhost:9090?file=path/to/fmu1.fmu"/>
-    <Simulator name="FMU2" source="fmu-proxy://localhost:9090?http://example.com/fmu2.fmu"/>
-    <Simulator name="FMU3" source="fmu-proxy://localhost:9090?guid=<fmu-guid-from-modelDescriptiongoes-here>"/>
+	<Simulator name="FMU1" source="proxyfmu://localhost?file=path/to/fmu1.fmu"/>
+	<Simulator name="FMU1" source="proxyfmu://127.0.0.1:9090?file=path/to/fmu2.fmu"/>
```

Comment:
Only relative paths are supported right?

Reply:
Absolute paths should be ok if they start with `file::///`. Document I guess?

Reply:
ordinary absolute path with `file:///` is actually not shown in the doc either. That should be added to the Simulation configurations section (https://open-simulation-platform.github.io/libcosim/configuration#simulator)

But I was thinking about absolute path with proxyfmu, is that possible?

Reply:
Mhh?
One could either do:
`<Simulator name="FMU1" source="proxyfmu://localhost?file=path/to/fmu1.fmu"/>`
or
`<Simulator name="FMU1" source="proxyfmu://localhost?file=file:///C:/path/to/fmu1.fmu"/>`

Reply:
My mistake, it works and should be documented as an example here in the user guide and in the sim configuration ([open-simulation-platform.github.io/libcosim/configuration#simulator](https://open-simulation-platform.github.io/libcosim/configuration#simulator))

Reply:
I added documentation for absolute paths here, but I don't think it is necessary also for the configuration page. I don't think it should be encouraged either.